docs/getting-started.md

```diff
@@ -0,0 +1,316 @@
+## Getting Started
+
+This document introduces how to create a kubernetes cluster locally using kind and how to deploy the basic skywalking components to the cluster.
+
+### Prerequisites
+- [docker](https://docs.docker.com/get-docker/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- [swctl](https://github.com/apache/skywalking-cli?tab=readme-ov-file#install)
+
+### Step1: Create a kubernetes cluster locally using kind
+
+> Note: If you have a kubernetes cluster already, you can skip this step.
```

Comment:
What's the required version?

Reply:
Updated. It should be more than v1.21.10